### ISSUE TITLE
feat: add support for graphql in Dart.

### ISSUE THREAD

**Issue by 0p3r4t0r** [graphql/graphiql#3476]
- [x] Does it currently support dart? --> checked Issued --> No: #2981
- [x] Any similar issues? --> Support for Python: #2556 

I would like to open a PR to add support for syntax highlighting in Dart if that's alright & appropriate.

Let me know what you think!

**Comment by acao** [comment]
yes please do! you can use our custom snapshot setup as a TDD guide
